browser-extension/addTournesolRecommendations.js

```diff
@@ -78,6 +80,18 @@ const getTournesolComponent = (data) => {
     tournesol_link.append('learn more');
     inline_div.append(tournesol_link);
 
+    // Refresh button
+    refresh_button = document.createElement('img');
+    refresh_button.setAttribute('id', 'refresh_button');
+    refresh_button.setAttribute('src', chrome.runtime.getURL('images/Refresh_icon.png'));
+    refresh_button.setAttribute('width', '24');
+    refresh_button.className = 'icon';
+    refresh_button.onclick = () => {
+      refresh_button.className = 'icon-disabled';
```

Comment:
By using a `<button>`, you could simply set `refresh_button.disabled = true`. And that it can be styled with [`.icon:disabled`](https://developer.mozilla.org/fr/docs/Web/CSS/:disabled)

The advantage is that disabled buttons can't trigger click events: no double clicks, no duplicated requests to the API.